relocations/quarkus-tika/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-relocations-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-tika</artifactId>
+
+    <distributionManagement>
+        <relocation>
+            <groupId>io.quarkiverse.tika</groupId>
+            <artifactId>quarkus-tika</artifactId>
+            <version>1.0.1</version>
+            <message>${project.groupId}:${project.artifactId}:${project.version} was relocated to io.quarkiverse.tika:${project.artifactId}:1.0.1 and is not managed by io.quarkus:quarkus-bom:${project.version} anymore. Please, update the groupId and add the corresponding version to the dependency declaration in your project configuration. For more information about this change, please refer to https://github.com/quarkusio/quarkus/wiki/Migration-Guide-2.6</message>
```

Comment:
I just noticed it's saying it's not managed by `io.quarkus:quarkus-bom` anymore. It could have been `io.quarkus.platform:quarkus-bom` or simply `quarkus-bom`. Although it's not a big deal, I guess.

Reply:
Ah ah, I had an hesitation when doing it. I can fix that for sure but I'll do it as a follow up commit, tired of rebasing one by one :).